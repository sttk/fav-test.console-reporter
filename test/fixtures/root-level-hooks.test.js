'use strict';

require('./root-level-hooks-1');
require('./root-level-hooks-2');

