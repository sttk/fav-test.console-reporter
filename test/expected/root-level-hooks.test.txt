

before every test in every file
test run
  ✓ test

  suite1
before every test in every file
test1 run
    ✓ test1
    suite2
before every test in every file
test2 run
      ✓ test2

before every test in every file
test3 run
  ✓ test3

  suite4
before every test in every file
test4 run
    ✓ test4
    suite5
before every test in every file
test5 run
      ✓ test5


  6 passing (5ms)

